app/scenes/image_preview/image_preview.js

```diff
@@ -0,0 +1,318 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {PropTypes, PureComponent} from 'react';
+import {
+    Animated,
+    Dimensions,
+    InteractionManager,
+    PanResponder,
+    Platform,
+    ScrollView,
+    StyleSheet,
+    Text,
+    TouchableOpacity,
+    TouchableWithoutFeedback,
+    View
+} from 'react-native';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import LinearGradient from 'react-native-linear-gradient';
+
+import FileAttachmentIcon from 'app/components/file_attachment_list/file_attachment_icon';
+import FileAttachmentPreview from 'app/components/file_attachment_list/file_attachment_preview';
+
+const {View: AnimatedView} = Animated;
+const {height: deviceHeight, width: deviceWidth} = Dimensions.get('window');
```

Comment:
again, I think we should be careful with the Dimensions here as they will change when the device rotates